lib/src/room.dart

```diff
@@ -159,10 +160,16 @@ class KurentoRoom extends ChangeNotifier implements Room {
   }
 
   // The audio is muted if the first audio track is disabled.
-  bool get _isAudioMuted => _localStream!.getAudioTracks().isEmpty ? false : !_localStream!.getAudioTracks()[0].enabled;
+  bool get _isAudioMuted => _localStream!.getAudioTracks().isEmpty ? true : !_localStream!.getAudioTracks()[0].enabled;
 
   // The video is muted if the first video track is disabled.
-  bool get _isVideoMuted => _localStream!.getVideoTracks().isEmpty ? false : !_localStream!.getVideoTracks()[0].enabled;
+  bool get _isVideoMuted {
+    MediaStreamTrack? activeVideoTrack = _activeVideoTrack;
+    return null == activeVideoTrack || !activeVideoTrack.enabled;
+  }
```

Comment:
This allows the manage the mutes in a seamless way. This separates the WebCam and Screenshare mutes from each other.

Reply:
this was inspired from a conversation with Troy. If the Explorer wants to handle some personal stuff quickly and would like to hide his screen quickly, stop and start a new screen share is much longer to achieve. Mute would be super quick. This start to be an advanced feature... I thought we could have our Explorer test that out and let us know what they think.